spellbook/models/opensea/solana/opensea_solana_schema.yml

```diff
@@ -24,12 +24,14 @@ models:
         description: "UTC event block time"
       - name: token_symbol
         description: "Token Symbol"
+      - name: token_address
+        description: "Token Address"
       - name: amount
         description: "Value of the trade at time of execution in the original currency"
       - name: amount_usd
         description:  "USD value of the trade at time of execution"
-      - name: trader
-        description:  "Trader"
+      - name: traders
+        description:  "Traders"
```

Comment:
if the description is just the same as the name of the column, I think we can just skip adding a description.